details/impersonation.md

```diff
@@ -33,9 +33,11 @@ Before an admin will be able to impersonate a user, they have to configure ROR p
 
 When you call Elasticsearch directly or through ROR Kibana, ROR ACL is defined by Settings (we can assume they are Main Settings). The Test Settings define another ACL, that is taken into consideration by ROR ES only when a proper impersonation header is passed. The header is managed by ROR internally. The Test Settings are active only for a strictly defined amount of time (by default it's *30 minutes*, but the admin can change it before applying Test Settings). After the time has expired, they are automatically invalidated (for security reasons). Obviously, the admin is allowed to invalidate the configured Test Settings in any time. There is no way to have more than one Test Settings configured at time.
 
-Detailed instructions on how to define Test Settings can be found [here](../examples/impersonation/test-settings-ui.md).
+ROR Kibana plugin provides dedicated UI to easily manage Test Settings. The [detailed instruction](../examples/impersonation/test-settings-ui.md) describes how to do that.
```

Comment:
The idea is to avoid pronouns. We could simplify  the above text as such:

See how to  [manage Test Settings using ROR Kibana plugin](...) with the dedicated UI.

Reply:
Yeah good point! This should be better:

"ROR Kibana plugin provides a dedicated Test Settings UI. See our [Test Settings management guide](....) for more information."